CHANGELOG

```diff
@@ -8,6 +8,46 @@ devel
 
 * Added compression property to ArangoSearch View storedValues
 
+* Added overwrite mode "ignore" for document inserts. This mode allows 
+  ignoring primary key conflicts on insert when the target document already
+  exists.
+  
+  The following overwrite modes now exist:
+
+  * "ignore": if a document with the specified `_key` value already exists,
+    nothing will be done, and no write operation will be carried out. The 
+    insert operation will return success in this case. This mode does not 
+    support returning the old or new document versions using the `returnOld`
+    and `returnNew` attributes.
+  * "replace": if a document with the specified `_key` value already exists,
+    it will be overwritten with the specified document value. This mode will
+    also be used when no overwrite mode is specified but the `overwrite`
+    flag is set to `true`.
+  * "update": if a document with the specified `_key` value already exists,
+    it will be patched (partially updated) with the specified document value. 
+  * "conflict": if a document with the specified `_key` value already exists,
+    return a unique constraint violation error so that the insert operation
+    fails. This is also the default behavior in case the overwrite mode is
+    not set.
+
+  The overwrite mode "ignore" can also be used from AQL INSERT operations
+  by specifying it in the INSERT's `OPTIONS`, e.g.
+
+      INSERT { _key: ..., .... } INTO collection OPTIONS { overwriteMode: "ignore" }
+
+  Again, when the overwrite mode "ignore" is used from AQL, it does not
+  support returning the old or new document versions. Using "RETURN OLD" in 
+  an INSERT operation that uses the "ignore" overwrite mode will trigger
+  a parse error, as there will be no old version returned, and "RETURN NEW"
+  will only return the document in case it was inserted. In case the document
+  already existed, "RETURN NEW" will return "null".
```

Comment:
If AQL either returns the new document or null in the "ignore" case, why does the API not do the same for `returnNew`?

Reply:
@Simran-B : Doesn't it? Can you elaborate what the difference is?